public/js/components/Frames.js

```diff
@@ -66,6 +66,9 @@ const Frames = React.createClass({
 
   renderFrames() {
     let { frames } = this.props;
+    if (!frames) {
+      return null;
+    }
```

Comment:
Oh, I just pushed the commit that refactored it to `renderFrames(frames)` about 10 seconds after you clicked Merge. Do you want me to PR that one too?